Blocks/MovingBlock36.md

```diff
@@ -1,38 +1,39 @@
 ---
-title: Moving piston block 36
+title: Moving block 36
```

Comment:
perhaps `Moving block (block 36)` is a better title?

Reply:
I added that.